go.mod

```diff
@@ -81,38 +81,38 @@ require (
 	github.com/digitorus/pkcs7 v0.0.0-20230818184609-3a137a874352
 	github.com/distribution/reference v0.6.0
 	github.com/dustin/go-humanize v1.0.1
-	github.com/elastic/go-elasticsearch/v8 v8.13.1
+	github.com/elastic/go-elasticsearch/v8 v8.14.0
 	github.com/elimity-com/scim v0.0.0-20240320110924-172bf2aee9c8
 	github.com/evanphx/json-patch v5.9.0+incompatible
-	github.com/fsouza/fake-gcs-server v1.48.0
-	github.com/fxamacker/cbor/v2 v2.6.0
+	github.com/fsouza/fake-gcs-server v1.49.2
+	github.com/fxamacker/cbor/v2 v2.7.0
 	github.com/ghodss/yaml v1.0.0
 	github.com/gizak/termui/v3 v3.1.0
 	github.com/go-jose/go-jose/v3 v3.0.3
 	github.com/go-ldap/ldap/v3 v3.4.8
 	github.com/go-logr/logr v1.4.2
 	github.com/go-mysql-org/go-mysql v1.5.0 // replaced
 	github.com/go-piv/piv-go v1.11.0
-	github.com/go-resty/resty/v2 v2.12.0
+	github.com/go-resty/resty/v2 v2.13.1
 	github.com/go-webauthn/webauthn v0.10.2
 	github.com/gobwas/ws v1.4.0
 	github.com/gocql/gocql v1.6.0
-	github.com/gofrs/flock v0.8.1
+	github.com/gofrs/flock v0.12.0
```

Comment:
Flurry of releases, but the release logs seem fine: https://github.com/gofrs/flock/releases.